tests/test_PIAF.py

```diff
@@ -49,3 +49,13 @@ def test_children(self):
         with PI.PIAFDatabase() as db:
             children = db.children
         assert isinstance(children, dict)
+
+
+class TestDatabaseSearch:
+    """Test retrieving attributes"""
+
+    def test_search(self):
+        """Test that calling attributes on the database returns a list of attributes"""
+        with PI.PIAFDatabase() as db:
+            attributes = db.search([r'', r''])
+        assert isinstance(attributes, List)
```

Comment:
In this case you could actually use `isinstance(attributes, list)`, because `list` is a valid class. Only for the type hints you need `typing.List[Something]` for older versions of python. In python 3.10 they implemented `list.__class_getitem__` which allows `list[Something]` for type hints as well.

Reply:
ah yes i see. changed.